drivers/sensor/bmi08x/Kconfig

```diff
@@ -0,0 +1,93 @@
+# Bosch BMI08X inertial measurement configuration options
+
+# Copyright (c) 2022 Meta Platforms, Inc. and its affiliates
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMI08X
+	bool "Bosch BMI08X inertial measurement unit"
+	default y
+	depends on DT_HAS_BOSCH_BMI08X_ACCEL_ENABLED || DT_HAS_BOSCH_BMI08X_GYRO_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BMI08X_ACCEL),i2c) || $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BMI08X_GYRO),i2c)
+	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BMI08X_ACCEL),spi) || $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BMI08X_GYRO),spi)
```

Comment:
Not sure why this didn't get flagged by the compliance check, but will you please shorten these long lines?